### PR TITLE
Update github-workflow.json

### PR DESCRIPTION
**Pull request by LeonardLaszlo**
/ character should be escaped in regular expressions

Closes #000

### Types of changes

  - [ x ] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to change)

It fixes a syntax error in Itellij Idea when the path to an action is in the following format: ./path/to/localfile.yml

Try to match the previous regexp ( ^(.+/)+(.+)\.(ya?ml)(@.+)?$ ) against a valid input ( ./path/to/localfile.yml ) here:
https://regex101.com/

Use the example above on https://regex101.com/

### CODE REVIEW

**Comment by YunaBraska** [comment]
Hey @LeonardLaszlo thx for creating a PR. 
These schema files are coming from https://json.schemastore.org/github-workflow.json
Probably you can also solve this globally

**Comment by LeonardLaszlo** [comment]
Done here:
https://github.com/SchemaStore/schemastore/pull/3943

**Comment by YunaBraska** [comment]
Next version with your change is published and will be available as soon as it's approved (usually takes 1 day). Thanks for the contribution.

File: src/main/resources/schemas/github-workflow.json [hunk 1]
```diff
@@ -491,7 +491,7 @@
           "$comment": "https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iduses",
           "description": "The location and version of a reusable workflow file to run as a job, of the form './{path/to}/{localfile}.yml' or '{owner}/{repo}/{path}/{filename}@{ref}'. {ref} can be a SHA, a release tag, or a branch name. Using the commit SHA is the safest for stability and security.",
           "type": "string",
-          "pattern": "^(.+/)+(.+)\\.(ya?ml)(@.+)?$"
+          "pattern": "^(.+\/)+(.+)\\.(ya?ml)(@.+)?$"
```
Comment (LecrisUT) [inline src/main/resources/schemas/github-workflow.json:494]: See comment https://github.com/SchemaStore/schemastore/pull/3943#issuecomment-2253076172

**Pull request merged**